core/ui/public/i18n/en/translation.json

```
@@ -1034,7 +1037,27 @@
     "understood_remove_node": "I understand, remove node",
     "validation_node_is_leader": "Cannot remove leader node",
     "validation_node_in_use": "The node cannot be removed because the following app is installed on it: <strong>{apps}</strong> | The node cannot be removed because the following apps are installed on it: <strong>{apps}</strong>",
-    "validation_node_not_found": "Node not found"
+    "validation_node_not_found": "Node not found",
+    "remove_node_apps_confirm": "The following app is installed on {node}: <strong>{nodeApps}</strong>.<br>If you remove the node, <strong>{nodeApps} will be removed as well</strong>. | The following apps are installed on {node}: <strong>{nodeApps}</strong>.<br>If you remove the node, <strong>these apps will be removed as well</strong>.",
+    "promote_to_leader": "Promote to leader",
+    "promote_node_to_leader": "Promote node to leader",
+    "promote_node_confirm": "<strong>{name}</strong> will be promoted to leader role.",
+    "type_to_confirm_promotion": "Type <strong>{name}</strong> to confirm node promotion",
+    "node_vpn_endpoint_address": "{node} VPN endpoint address",
+    "node_vpn_endpoint_address_tooltip": "Make sure this VPN endpoint address is reachable by all other cluster nodes",
+    "node_vpn_endpoint_port": "{node} VPN endpoint port",
+    "understood_promote_node": "I understand, promote node",
+    "reset_backup_encryption_password_info": "After node promotion, go to <strong>Backup</strong> section and reset backup encryption password",
+    "new_cluster_admin_message": "From now on, cluster administration will be available at:<br><code>https://{endpointAddress}/cluster-admin/</code>",
+    "check_node_connectivity": "Check node connectivity",
+    "cannot_promote_node": "Cannot promote node",
+    "validation_node_is_already_leader": "Node already has leader role",
+    "validation_endpoint_address_not_reachable": "VPN endpoint address is not reachable, please check it again. If you know what you are doing and want to skip this check, deselect <strong>Check node connectivity</strong> and try again",
```

This is a check box, maybe `uncheck` is better?
```suggestion
    "validation_endpoint_address_not_reachable": "VPN endpoint address is not reachable, please check it again. If you know what you are doing and want to skip this check, uncheck <strong>Check node connectivity</strong> and try again",
```